tests/unit/lib/SGSparseMatrix_unittest.cc

```diff
@@ -157,4 +177,101 @@ TEST(SGSparseMatrix, io_libsvm)
 			EXPECT_NEAR(m[i].features[j].entry, m_from_file[i].features[j].entry, 1E-14);
 		}
 	}
+
+    //remove file generated by test
+	unlink("SGSparseMatrix_io_libsvm_output.txt");
```

Comment:
not sure why the indentation is like this, it appears fine before the pull reques